docs/modules/ROOT/pages/tls.adoc

```diff
@@ -0,0 +1,38 @@
+= Configuring TLS
+:description: Hazelcast TLS is a security feature that provides encryption and authentication of data transmitted between Hazelcast nodes, preventing unauthorized access and eavesdropping.
+
+{description}
+
+Hazelcast TLS uses standard SSL/TLS certificates, which can be obtained from trusted Certificate Authorities (CA) or self-signed. The certificates are used to establish trust between the communicating parties and ensure the authenticity of the communication channel.
+
+== Configuring Hazelcast TLS
+
+Below are the configuration options for the TLS.
+
+[cols="25%m,75%a"]
+|===
+|Field|Description
+
+|`secretName`
```

Comment:
@dzeromski-hazelcast Could you provide a sample secret yaml and one-line `kubectl create secret` command?